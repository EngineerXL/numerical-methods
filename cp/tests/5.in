5216225124368878122267204
1511449792231214859941636
