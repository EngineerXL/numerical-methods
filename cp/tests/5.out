7884062380458583730729109034459977252723436905744
